### ISSUE TITLE
Document client-side uploading

### ISSUE THREAD

**Issue by skorokithakis** [klis87/django-cloudinary-storage#10]
Hello!

I am currently using django-cloudinary-storage with the naive, server-uploading method, where the user uploads the photo to the server and the server uploads it to Cloudinary. That works fine, but soon I will switch to client-uploaded images, where the user uploads the image directly to Cloudinary and then sends me the URL (or whatever Cloudinary returns).

Could some documentation be added to detail how I can support this method? What do I need to give Django to create these fields and how?

**Comment by skorokithakis** [comment]
It seems like the way to do this is to just assign the final Cloudinary path to the `ImageField`:

```python
my_model.image = "media/imagefile"
my_model.save()
```

**Comment by klis87** [comment]
@skorokithakis Exactly. If you use any Django storage, file upload must go through Django. If you want to upload directly to cloudinary, then you actually won't use Django storage for uploading at all. However, you will still benefit from it, for example by utilizing `delete-orphaned-media` command to clean unnecessary files.

Is everything working fine with your method? Do you think we should add this to documentation?

**Comment by skorokithakis** [comment]
Yes, everything seems to be working great so far. I think it should be added, yes, as I imagine many people will be using client-side uploading and the fix is so simple that I don't see why not.
